parlai/mturk/core/dev/agents.py

```diff
@@ -161,15 +159,16 @@ def __init__(self, opt, mturk_manager, hit_id, assignment_id, worker_id):
         self.worker_id = worker_id
         self.some_agent_disconnected = False
         self.hit_is_expired = False
-        self.hit_is_abandoned = False  # state from Amazon MTurk system
-        self.hit_is_returned = False  # state from Amazon MTurk system
-        self.hit_is_complete = False  # state from Amazon MTurk system
+        self.hit_is_abandoned = False  # state from MTurk SNS system
```

Comment:
what is SNS

Reply:
Simple Notification Service. It's now the only place we get pinged about someone who lets a HIT expire without completing it.